src/providers/ipa/ipa_sudo.c

```diff
@@ -250,16 +250,16 @@ ipa_sudo_init_ipa_schema(TALLOC_CTX *mem_ctx,
                          CONFDB_SUDO_THRESHOLD, CONFDB_DEFAULT_SUDO_THRESHOLD,
                          &sudo_ctx->sudocmd_threshold);
     if (ret != EOK) {
-        DEBUG(SSSDBG_OP_FAILURE, "Could not parse sudo search base\n");
-        return ret;
+        DEBUG(SSSDBG_CRIT_FAILURE, "Could not get sudo threshold\n");
+        goto done;
```

Comment:
This affects function execution flow. If this fix is consistent with this PR topic focused on mainly on logs?

Reply:
No, this doesn't change execution flow.
This fixes minor mem leak, isn't worth stand alone PR.